src/pbview.cpp

```diff
@@ -120,6 +120,10 @@ void PbView::run(bool auto_download, bool wrap_scroll)
 			continue;
 		}
 
+		if (strcmp(event, "RESIZE") == 0) {
+			handle_resize();
```

Comment:
I think there should be a `continue` here, because the following lines (`keys->get_operation(event, "podboat")`...) make no sense for the resize even.

Reply:
Done